src/tngcli/cli.py

```diff
@@ -684,6 +692,54 @@ def parse_args(args):
                             default=False,
                             help='Returns detailed info on specified request')
 
+    parser_req.add_argument('--scale_in',
+                            action='store_true',
+                            required=False,
+                            default=False,
+                            help='Scale in a service')
+
+    parser_req.add_argument('--scale_out',
+                            action='store_true',
+                            required=False,
+                            default=False,
+                            help='Scale out a service')
+
+    parser_req.add_argument('--vnfd_uuid',
+                            metavar='vnfd_uuid',
+                            required=False,
+                            default=False,
+                            help='Specify the vnf descriptor uuid')
+
+    parser_req.add_argument('--vnf_uuid',
+                            metavar='vnfd_uuid',
+                            required=False,
+                            default=False,
+                            help='Specify the vnf instance uuid')
+
+    parser_req.add_argument('--service_uuid',
```

Comment:
What is this for? Isn't it the same as instance_uuid?

Reply:
Yes, the instantiation if from services, not from here. Removed